docs/source/json-api/index.rst

```diff
@@ -12,22 +12,22 @@ tracker
 <https://github.com/digital-asset/daml/issues/new?milestone=HTTP+JSON+API+Maintenance>`_
 or `on Slack <https://hub.daml.com/slack/>`_.
 
-The JSON API provides a significantly simpler way than :doc:`the Ledger
-API </app-dev/index>` to access *basic active contract set
-functionality*:
+The **JSON API** provides a significantly simpler way than :doc:`the Ledger
+API </app-dev/index>` to interact with a ledger by providing *basic active contract set functionality*:
 
-- creating contracts,
-- exercising choices on contracts, and
-- querying the current active contract set.
+- create contract,
+- fetch active contract by contract ID or contract key,
```

Comment:
Keep the description here informal, describing categories of functionality not enumerating features; fetch is already covered by "query the current active contract set".